src/pages/patientView/PatientViewPageTabs.tsx

```diff
@@ -275,7 +288,7 @@ export function tabs(
                                 }
                                 sampleIds={
                                     sampleManager
-                                        ? sampleManager.getSampleIdsInOrder()
+                                        ? sampleManager.getActiveSampleIdsInOrder()
```

Comment:
@gblaih this was the big change.  It turns out the SampleManger is the "one" that knows the about the ordering of samples (based on a clinical data). However, the getSampleIdsInOrder doesn't know about which sample is "active" in the page (sample mode or patient mode in which all are active).   In fact, it DOES have the information (sampleIds collection passed at construction), so it was easy to add this.  Now, we can get the ordered list of active sample ids, which is what the tables need.